src/app/invest/invest.component.scss

```diff
@@ -127,22 +98,18 @@ p {
   }
 
   &__return-label p {
-    padding-bottom: 0;
-    padding-left: 4px;
-    padding-top: 32px;
```

Comment:
When you have multiple paddings (top, right, bottom) try to use shorthand version (padding: value value value value). This shorter version uses clockwise direction to define paddings so when you write it like this, first value is padding-top, second is padding-right, third is padding- bottom and last is padding-left.
This is applied to margins as well.
Much leaner and cleaner.

Reply:
Yes I know, I'm familiar with that. ;-) I've used this shorthand version in some tasks before.